RUFAS/routines/EEE/emissions.py

```diff
@@ -509,25 +506,114 @@ def _calculate_emissions_by_field(
-                fertilizer_applications["phosphorus"]
-                * fraction_of_total_mass_grown
-                * EMBEDDED_PHOSPHORUS_FERTILIZER_EMISSIONS_FACTOR
+                fertilizer_application["phosphorus"] * split_factor * EMBEDDED_PHOSPHORUS_FERTILIZER_EMISSIONS_FACTOR
             )
-            crop["potassium_fertilizer_used"] = fertilizer_applications["potassium"] * fraction_of_total_mass_grown
+            crop["potassium_fertilizer_used"] += fertilizer_application["potassium"] * split_factor
             crop["potassium_fertilizer_embedded_CO2_emissions"] = (
-                fertilizer_applications["potassium"]
-                * fraction_of_total_mass_grown
-                * EMBEDDED_POTASSIUM_FERTILIZER_EMISSIONS_FACTOR
+                fertilizer_application["potassium"] * split_factor * EMBEDDED_POTASSIUM_FERTILIZER_EMISSIONS_FACTOR
             )
-            crop["manure_nitrogen_used"] = manure_applications["nitrogen"] * fraction_of_total_mass_grown
-            crop["manure_nitrogen_requested"] = manure_requests["nitrogen"] * fraction_of_total_mass_grown
 
-        return feeds_grown
+    def _apply_fertilizer_to_next_crop(
+        self,
+        fertilizer_application: dict[str, Any],
+        sorted_crops: list[dict[str, Any]],
+        fertilizer_application_date: datetime.date,
+    ) -> bool:
+        """
+        Applies fertilizer to the next available crop after harvest and before the next planting.
+        Returns True if fertilizer was applied, False otherwise.
+        """
+        for i, crop in enumerate(sorted_crops):
+            crop_harvest_date = Time.convert_year_jday_to_date(crop["harvest_year"], crop["harvest_day"])
+            if i + 1 < len(sorted_crops):
+                next_crop = sorted_crops[i + 1]
+                next_crop_planting_date = Time.convert_year_jday_to_date(
+                    next_crop["planting_year"], next_crop["planting_day"]
+                )
+                if crop_harvest_date < fertilizer_application_date < next_crop_planting_date:
+                    next_crop["nitrogen_fertilizer_used"] += fertilizer_application["nitrogen"]
+                    next_crop["nitrogen_fertilizer_embedded_CO2_emissions"] = (
+                        fertilizer_application["nitrogen"] * EMBEDDED_NITROGEN_FERTILIZER_EMISSIONS_FACTOR
+                    )
+                    next_crop["phosphorus_fertilizer_used"] += fertilizer_application["phosphorus"]
+                    next_crop["phosphorus_fertilizer_embedded_CO2_emissions"] = (
+                        fertilizer_application["phosphorus"] * EMBEDDED_PHOSPHORUS_FERTILIZER_EMISSIONS_FACTOR
+                    )
+                    next_crop["potassium_fertilizer_used"] += fertilizer_application["potassium"]
+                    next_crop["potassium_fertilizer_embedded_CO2_emissions"] = (
+                        fertilizer_application["potassium"] * EMBEDDED_POTASSIUM_FERTILIZER_EMISSIONS_FACTOR
+                    )
```

Comment:
```suggestion
                    next_crop["nitrogen_fertilizer_used"] += fertilizer_application["nitrogen"]
                    next_crop["nitrogen_fertilizer_embedded_CO2_emissions"] += (
                        fertilizer_application["nitrogen"] * EMBEDDED_NITROGEN_FERTILIZER_EMISSIONS_FACTOR
                    )
                    next_crop["phosphorus_fertilizer_used"] += fertilizer_application["phosphorus"]
                    next_crop["phosphorus_fertilizer_embedded_CO2_emissions"] += (
                        fertilizer_application["phosphorus"] * EMBEDDED_PHOSPHORUS_FERTILIZER_EMISSIONS_FACTOR
                    )
                    next_crop["potassium_fertilizer_used"] += fertilizer_application["potassium"]
                    next_crop["potassium_fertilizer_embedded_CO2_emissions"] += (
                        fertilizer_application["potassium"] * EMBEDDED_POTASSIUM_FERTILIZER_EMISSIONS_FACTOR
                    )
```
The emissions should be increased in the same fashion as the nutrient amounts, not overwritten